### PR TITLE
Obtusely simple prototype.

### PR DESCRIPTION
**Pull request by octocrux**

### CODE REVIEW

File: fastCheck.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+(function () {
```
Comment (marvelousNinja) [inline fastCheck.js]: What the heck is this file?

File: first.js [hunk 2]
```diff
@@ -1,60 +1,38 @@
-//(function (a, b) {} ('a', 'b'))
```
Comment (marvelousNinja) [inline first.js]: Dead code, WTF?

File: fastCheck.js (hunk 1 above)
Comment (octocrux) [inline fastCheck.js]: O-oh.

**Comment by marvelousNinja** [comment]
Fixes #1.

File: first.js [hunk 3]
```diff
@@ -1,60 +1,39 @@
...
-  var attacker = [new Ship({name: 'hlam', slots: 6}), new Ship({name: 'hlam2', defaultState: 22 }),
-    new Ship({name: 'hlam3', slots: 1 })];
-  var defender = [new Ship({name: 'hlam444', slots: 3 })];
-
-  simulateBattle(attacker, defender);
-  console.dir(upgrades);
-
-}());
\ No newline at end of file
+  console.log('Attacker\'s chances are ' + attackerWins / simulations + '%');
```
Comment (octocrux) [inline first.js]: there *is* a newline at end of file.

**Comment by marvelousNinja** [comment]
Looks fine to me.

**Pull request merged**